app/scripts/modules/core/cluster/filter/clusterFilter.component.ts

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+import {CLUSTER_FILTER_SERVICE} from 'core/cluster/filter/clusterFilter.service';
+
+import {compact, uniq, map} from 'lodash';
+import {IScope, module} from 'angular';
+import {Application} from '../../application/application.model';
+
+export const CLUSTER_FILTER = 'cluster';
```

Comment:
we want module names to basically follow directory structure, prefixed with `spinnaker`, e.g. `spinnaker.core.cluster.filter.component`